e2e/tests/workspace/workspace-create.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('Workspace create:', () => {
     await createButton.click();
     await (new DataPage(page)).waitUntilPageReady();
 
-    console.log("done");
+    console.log('done');
```

Comment:
nit: this log message is probably unnecessary, I'd just remove it.

Reply:
Done